docs/v1/user-story

```diff
@@ -0,0 +1,124 @@
+A Year In The Life Of A Kubernetes Service Developer
+
+0. Today
```

Comment:
I think I got the markdown correct. Having trouble previewing the result for some reason...